symplyphysics/laws/kinematic/rotational_inertia_cartesian_integral.py

```diff
@@ -0,0 +1,49 @@
+from sympy import Integral
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    Function,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## In case of a rigid body with a continuously distributed mass, its rotational inertia is expressed
+## as an integral over the entire body.
+
+# Law: I = Integral(rho(x, y, z) * (x**2 + y**2), dx*dy*dz)
+## I - rotational inertia
+## r - perpendicular distance from rotation axis to volume element
+## rho - density of the volume element
+## x**2 + y**2 = r**2 - square of perpendicular distance to rotation axis
+## dx*dy*dz = dV - volume element
+
+# Conditions:
+## 1. The z axis is the rotational axis of the body
+## 2. The integration is carried out over the entire body as to include every volume element.
+
+density = Function("density", units.mass / units.length**3)
+x = Symbol("x", units.length)
+y = Symbol("y", units.length)
+z = Symbol("z", units.length)
+
+
+def rotational_inertia_law(x_limits, y_limits, z_limits):
```

Comment:
You can describe law as sympy `Integral` without helper function, ie `law = Integral(density * r**2, (x, x_start, x_end), (y, y_start, y_end), (z, z_start, z_end)`
But if you want to support non cartesian coordinates, you will have to use helper function, like you did.